pkg/proto/transactions_with_sig.go

```diff
@@ -358,7 +358,7 @@ func NewUnsignedTransferWithSig(senderPK crypto.PublicKey, amountAsset, feeAsset
 }
 
 func (tx *TransferWithSig) BodyMarshalBinary(Scheme) ([]byte, error) {
-	b, err := tx.Transfer.marshalBinary()
+	b, err := tx.marshalBinary()
```

Comment:
Here the same

Reply:
See this https://github.com/wavesplatform/gowaves/pull/1644#discussion_r2017664220